client/app/dbaas/logs/detail/options/logs-options.less

```diff
@@ -0,0 +1,43 @@
+.log-options {    
+    .log_options_back_button {
+        button {
+            padding-top: 1.2rem;
+            padding-right: 0rem;
+        }
+    }
+
+    .total-price {
+        text-align: right;
+    }
+
+    .current-options-tile {
+        text-align: center;
+        margin-bottom: 2rem;
+        .cui-tile {
+            padding: 0rem;
+        }
+        .cui-tile__item:last-child::after {
+            background-color: transparent;
+        }
+
+        .oui-header_4 {
+            margin: 0em;
+        }
+    }
+
+    .log-quantity {
+        .oui-input-group {
+            margin: 0rem;
+        }
+    }
+
+    // ToDo: Remove once the components supports hiding the paging information
+    oui-pagination {
```

Comment:
We hide pagination cause it options won't be paginated ever?

Reply:
And I know it's a hack but maybe you could try to push a very large number into oui-datagrid page-size binding?  

page-size="9999999"

Reply:
The datagrid shows the pagination, even if there is one record and the page-size is 99999999! The data grid does not hide the pagination, unless the number of records is 0. Hence we will have to use css for now, until we have a property that we can configure, to hide pagination.